doc/lua_api.md

```diff
@@ -8910,6 +8910,10 @@ Used by `minetest.register_entity`.
     -- A table of object properties, see the `Object properties` section.
     -- The properties in this table are applied to the object
     -- once when it is spawned.
+	-- Contrary to custom fields down below, initial_properties is a table
+	-- shared between all the instances of an entity. Changing any of its
+	-- fields on one instance will alter any other present and future instance
+	-- as well
```

Comment:
I would get right to the point and reword this like so:

```suggestion
	-- Do not try to change this table at runtime, because it is shared
	-- between all instances of an entity.
```

Reply:
I think it's important to underline that this behaves differently from custom fields, which is what it took me some time to realise

Reply:
I would solve that by adding a paragraph at the top explaining how the entity definition table becomes a metatable of a newly created per-entity table.

Reply:
Wait, so if I have a custom field that is a table, is that shared as well?

Reply:
Should be fine now